index.html

```diff
@@ -13,9 +13,14 @@
 </head>
 
 <body>
+  
   <nav class="navbar navbar-expand-lg bg-body-tertiary">
     <div class="container-fluid">
-      <a class="navbar-brand" href="#">PulseZest</a>
+      <a href="#" class="navbar-brand">
+        <img src="images/logo.png" alt="Your Logo" class="logo">

```

Comment:
replace this img tag with give below source code.. Because this time this logo will disturb the NavBar. 
` <img src="images/logo.png" alt="Your Logo" class="logo" style="width: 70px;"> <span>PulseZest</span>`